lib/msal-core/src/telemetry/TelemetryManager.ts

```diff
@@ -55,7 +55,11 @@ export default class TelemetryManager {
             sdk: "msal.js",
             sdkVersion: libraryVersion(),
             applicationName,
-            applicationVersion
+            applicationVersion,
+            networkInformation: {
+                // @ts-ignore
+                connectionSpeed: navigator && navigator.connection && navigator.connection.effectiveType
```

Comment:
I see this is in two places...could it just be in one place, e.g. the constructor?

Reply:
Yeah the doubling there is unfortunate. TelemetryManager has a static 'get me a dummy instance' which would be used internally when devs don't set their own. 
I guess you could argue that it is a static value that should just be set by the Telemetry Manager constructor.... 
Infact I think I am convinced.
Will update

Reply:
ahhh wait, now I know why. its because this Telemetry platform object is passed to default event and so should the Telemetry Manager take an object and set another value on it ... I am not sure  that is ideal

Reply:
There is also wanting Telemetry manager to be something that can be in common ... maybe that is too soon?
Thoughts?

Reply:
resolved